desktop-app/app/components/UserPreferences/index.js

```diff
@@ -5,12 +5,46 @@ import Checkbox from '@material-ui/core/Checkbox';
 import TextField from '@material-ui/core/TextField';
 import Input from '@material-ui/core/Input';
 import SettingsIcon from '@material-ui/icons/Settings';
+import Select from 'react-select';
 
 import commonStyles from '../common.styles.css';
 import styles from './styles.module.css';
 import {DEVTOOLS_MODES} from '../../constants/previewerLayouts';
 import ScreenShotSavePreference from '../ScreenShotSavePreference/index';
 import {userPreferenceSettings} from '../../settings/userPreferenceSettings';
+import {notifyPermissionPreferenceChanged} from '../../utils/permissionUtils.js';
+const selectStyles = {
+  control: selectStyles => ({...selectStyles, backgroundColor: '#ffffff10'}),
+  option: (selectStyles, {data, isDisabled, isFocused, isSelected}) => {
+    const color = 'white';
+    return {
+      ...selectStyles,
+      backgroundColor: isDisabled
+        ? null
+        : isSelected
+        ? '#ffffff40'
+        : isFocused
+        ? '#ffffff20'
+        : null,
+      color: 'white',
+
+      ':active': {
+        ...selectStyles[':active'],
+        backgroundColor: !isDisabled && '#ffffff40',
+      },
+    };
+  },
+  input: selectStyles => ({...selectStyles}),
+  placeholder: selectStyles => ({...selectStyles}),
+  singleValue: (selectStyles, {data}) => ({...selectStyles, color: 'white'}),
+  menu: selectStyles => ({...selectStyles, background: '#4b4b4b', zIndex: 100}),
+};
+
+const permissionsOptions = [
+  {value: 'Allow always', label: 'Allow always'},
```

Comment:
Can you move the string values to a constants file and refer from there?

For a similar use-case in #425 PR, I have added these to  `src/contansts/values.js` file.

Reply:
done